integration/src/test/resources/org/mule/test/integration/exceptions/exception-propagation-mule-5737-config.xml

```diff
@@ -12,26 +12,38 @@
 
     <flow name="flowWithChildFlow">
         <flow-ref name="childFlow"/>
-        <test:exception-strategy
-                class="org.mule.test.integration.exceptions.ExceptionPropagationMule5737TestCase$SensingExceptionStrategy"/>
+        <error-handler>
+            <on-error-continue>
+                <custom-processor class="org.mule.test.integration.exceptions.ExceptionPropagationMule5737TestCase$SensingExceptionParentProcessor"/>
```

Comment:
Custom processor is going away too... this kind of configuration has to be done using the test plugin